client/sites-dashboard/index.ts

```diff
@@ -9,5 +9,5 @@ export default function () {
 		return;
 	}
 
-	page( '/sites-dashboard', sitesDashboard, makeLayout, clientRender );
+	page( '/sites-dashboard/:launchStatus?', sitesDashboard, makeLayout, clientRender );
```

Comment:
I think we need to revert this change.

Reply:
```suggestion
	page( '/sites-dashboard', sitesDashboard, makeLayout, clientRender );
```

Sure!

In the future, you're welcome to make this change yourself.